src/Corrade/Utility/Debug.cpp

```diff
@@ -117,7 +117,7 @@ CORRADE_VISIBILITY_EXPORT
     #endif
 #endif
 DebugGlobals debugGlobals{
-    &std::cout, &std::cerr, &std::cerr,
+    Debug::defaultOutput(), Warning::defaultOutput(), Error::defaultOutput(),
```

Comment:
```suggestion
    #if defined(CORRADE_TARGET_MINGW) && defined(CORRADE_TARGET_CLANG)
    /* Referencing the globals directly makes MinGW Clang segfault for some reason */
    Debug::defaultOutput(), Warning::defaultOutput(), Error::defaultOutput(),
    #else
    &std::cout, &std::cerr, &std::cerr,
    #endif
```

so it's documented why is it like this (so I don't revert this by accident), and used only when needed. I hope the macros do the right thing -- can you confirm? :)

Reply:
Nice, works as expected !